pkgs/development/python-modules/polarizationsolver/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage, lib, fetchFromGitLab, python
+, numpy, scipy, periodictable, fields
+}:
+
+buildPythonPackage rec {
+  pname = "polarizationsolver";
+  version = "00424ac4";
```

Comment:
best practice I've seen with this is have

``version = "unstable-2021-XX-XX"`` (i.e. the date of the commit), and specify the full git hash revision in ``src.rev``